server/src/controllers/Events/resolver.ts

```diff
@@ -710,17 +711,31 @@ ${venueDetails}`;
     });
     await deleteEventReminders(id);
 
+    const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter.id}`;
+    const eventURL = `${process.env.CLIENT_LOCATION}/events/${event.id}?emaillink=true`;
     const notCanceledRsvps = event.event_users;
 
     if (notCanceledRsvps.length) {
       const emailList = notCanceledRsvps.map(({ user }) => user.email);
-      const subject = `Event ${event.name} canceled`;
-      const body = `The event ${event.name} was canceled`;
+      const subject = `Event ${event.name} is canceled`;
+      const cancelEventEmail = `The Upcoming Event for ${event.name} is canceled.<br />
+      <br />
+      View Upcoming Events for ${event.chapter.name}: <a href='${chapterURL}'>${event.chapter.name} chapter</a>.<br />
+      ----------------------------<br />
+      <br />
+      - Stop receiving upcoming event notifications for ${event.chapter.name}. you can do it here: <a href="${eventURL}">${eventURL}</a>.<br />
```

Comment:
From event page, only further event notifications for that event can be changed. Seems that either wording or url needs to be changed.

Reply:
I have updated it to this

```md
- Stop receiving event notifications for ${event.name}. You can do it here: <a href="${eventURL}">${eventURL}</a>.<br />
```

But it maybe redundant, should we delete that line?